Content.Server/GameObjects/Components/Items/Storage/ServerStorageComponent.cs

```diff
@@ -118,21 +119,58 @@ bool IAttackby.Attackby(IEntity user, IEntity attackwith)
         /// <returns></returns>
         bool IUse.UseEntity(IEntity user)
         {
-            SendNetworkMessage(new OpenStorageUIMessage());
+            var user_actor = user.GetComponent<BasicActorComponent>();
+            SubscribeActor(user_actor);
+            SendNetworkMessage(new OpenStorageUIMessage(), user_actor.playerSession.ConnectedClient);
+            UpdateClientInventory(user_actor);
             return false;
         }
 
         /// <summary>
-        /// Updates the storage UI on all clients telling them of the entities stored in this container
+        /// Updates the storage UI on all subscribed actors, informing them of the state of the container.
         /// </summary>
-        private void UpdateClientInventory()
+        private void UpdateClientInventories()
+        {
+            foreach (BasicActorComponent actor in SubscribedActors)
+            {
+                UpdateClientInventory(actor);
+            }
+        }
+
+        /// <summary>
+        /// Adds actor to the update list.
+        /// </summary>
+        /// <param name="actor"></param>
+        public void SubscribeActor(BasicActorComponent actor)
+        {
+            if (!SubscribedActors.Contains(actor))
+            {
+                Logger.GetSawmill("Storage").Info("Added actor with attached entity UID {0}.", actor.playerSession.AttachedEntityUid);
```

Comment:
You can do `Logger.InfoS("storage", ...)` here to get the sawmill.

Also that ought to be a debug message.